doc/source/train/user-guides/using-gpus.rst

```diff
@@ -8,11 +8,18 @@ be done in a few lines of code.
 The main interface for configuring scale and resources
 is the :class:`~ray.train.ScalingConfig`.
 
-Scaling Configurations in Train (``ScalingConfig``)
----------------------------------------------------
+:class:`~ray.train.ScalingConfig` overview
+------------------------------------------
+Ray Train's :class:`~ray.train.ScalingConfig` configures the number of training workers
+and the resources they should use.
 
-The scaling configuration specifies distributed training properties like the number of workers or the
-resources per worker.
+A *worker* here refers to a *training worker*. Technically, it's a
+:ref:`Ray Actor <actor-key-concept>` that runs your training loop.
+
+The :class:`Trainer <ray.train.trainer.BaseTrainer>` object you instantiate in the
+training script contains the settings to run your training. When you call
+:meth:`Trainer.fit() <ray.train.trainer.BaseTrainer.fit>`, it will be scheduled
+as a :ref:`Ray Actor <actor-key-concept>`. It can then also use resources.
```

Comment:
This feels a little strange now because of the new format.

I am thinking we can simplify this by just talking about workers and resources here. The Trainer part is more advanced, and also more confusing from an implementation perspective, so I feel like it would be better to contain it in the `Trainer resources` section instead.